frontend/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,17 +1,72 @@
-import { FC } from 'common/types/types';
-import { Button } from 'components/common/common';
+import { FC, UserSignInRequestDto } from 'common/types/types';
+import { AppRoute } from 'common/enums/enums';
+import { useAppForm } from 'hooks/hooks';
+import { getNameOf, getValidClasses } from 'helpers/helpers';
+import { userSignIn as userSignInValidationSchema } from 'validation-schemas/validation-schemas';
+import { Button, Input, Link } from 'components/common/common';
+import { DEFAULT_SIGN_IN_PAYLOAD } from './common';
+import authImage from 'assets/img/auth.png';
+import logo from 'assets/img/logo.svg';
+import styles from 'components/auth/styles.module.scss';
```

Comment:
```suggestion
import styles from './styles.module.scss';
```
you need to keep all styles near the component

Reply:
These styles are used both by sign-up and sign-in components, that's why they were created at a parent folder

Reply:
anyway. you can duplicate the styles. everything should be in the separate folders